dbt_subprojects/dex/models/trades/_schema.yml

```diff
@@ -110,6 +110,9 @@ models:
     data_tests:
       - dbt_utils.unique_combination_of_columns:
           combination_of_columns:
+            - block_month
+            - block_date
+            - block_number
```

Comment:
```suggestion
```

Reply:
lets remove from your PR to keep clean, i will handle this on my side

Reply:
fyi, this is the test that is running forever in CI. removing should exclude that test and speed it up.

Reply:
ohhh makes sense, let me do that.